examples/trusted_information_retrieval/backend/build.rs

```diff
@@ -18,15 +18,17 @@ use std::path::Path;
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
     let proto_path = Path::new("../../../examples/trusted_information_retrieval/proto");
-    let file_path = proto_path.join("trusted_information_retrieval.proto");
+    for file in ["trusted_information_retrieval.proto", "database.proto"].iter() {
+        let file_path = proto_path.join(file);
 
-    // Tell cargo to rerun this build script if the proto file has changed.
-    // https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargorerun-if-changedpath
-    println!("cargo:rerun-if-changed={}", file_path.display());
+        // Tell cargo to rerun this build script if the proto file has changed.
+        // https://doc.rust-lang.org/cargo/reference/build-scripts.html#cargorerun-if-changedpath
+        println!("cargo:rerun-if-changed={}", file_path.display());
 
-    tonic_build::configure()
-        .build_client(false)
-        .build_server(true)
-        .compile(&[file_path.as_path()], &[proto_path])?;
+        tonic_build::configure()
+            .build_client(false)
+            .build_server(true)
+            .compile(&[file_path.as_path()], &[proto_path])?;
+    }
```

Comment:
Why was this changed? Is it not possible to compile them at once any more? Please add enough comments for future maintainers to figure out what's going on here.